maat-court-data-api/src/main/java/gov/uk/courtdata/prosecutionconcluded/CalculateCrownCourtOutcome.java

```diff
@@ -0,0 +1,54 @@
+package gov.uk.courtdata.prosecutionconcluded;
+
+import gov.uk.courtdata.enums.CrownCourtTrialOutcome;
+import gov.uk.courtdata.enums.PleaTrialOutcome;
+import gov.uk.courtdata.enums.VerdictTrialOutcome;
+import gov.uk.courtdata.model.crowncourt.OffenceSummary;
+import gov.uk.courtdata.model.crowncourt.ProsecutionConcluded;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class CalculateCrownCourtOutcome {
+
+    public String calculate(ProsecutionConcluded prosecutionConcluded) {
+
+        if (prosecutionConcluded.isConcluded()) {
```

Comment:
This check should happen way up in the process. The calculators shouldn't even be called if the concluded flag is false.